static/js/components/capability/multi-level-switch.js

```diff
@@ -92,17 +92,47 @@ class MultiLevelSwitchCapability extends BaseComponent {
   }
 
   connectedCallback() {
+    this._upgradeProperty('min');
+    this._upgradeProperty('max');
+
     // Default to on=true to display level
     this.on = typeof this.dataset.on !== 'undefined' ? this.dataset.on : true;
     this.level =
-      typeof this.dataset.level !== 'undefined' ? this.dataset.level : false;
+      typeof this.dataset.level !== 'undefined' ? this.dataset.level : this.min;
+    this.unit =
+      typeof this.dataset.unit !== 'undefined' ? this.dataset.unit : '%';
```

Comment:
I think this should only default to percent if min is 0/undefined and max is 100/undefined. If a developer decides their thing has min 0 max 150 and no unit then the property could display 140%

Reply:
Fair point. I was trying to keep backwards compatibility with how things functioned before, but this is the proper thing to do.